src/views/tutorial-view/types.ts

```diff
@@ -41,7 +42,9 @@ interface TutorialData
 }
 interface TutorialViewProps {
 	layoutProps: TutorialSidebarSidecarProps
-	product: LearnProductData
+	product: Omit<LearnProductData, 'slug'> & {
+		slug: LearnClientProduct['slug'] | 'hcp'
+	}
```

Comment:
We need to do some cleanup around these types with the 'hcp' / 'cloud' mapping. This was a quick workaround as the current tutorial view didn't account for the product slug being 'hcp'.